android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java

```diff
@@ -713,7 +734,7 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 
 		if (key.equals(TiC.PROPERTY_CONTENT_OFFSET)) {
 			setContentOffset(newValue);
```

Comment:
Actually, there is no need to call it here. We will call `setContentOffset(int x, int y)` later from `onScrollChanged` which is triggered by the `scrollTo` on the next line.